services/forms/user_form.go

```diff
@@ -118,7 +119,8 @@ func IsEmailDomainListed(list []string, email string) bool {
 	domain := strings.ToLower(email[n+1:])
 
 	for _, v := range list {
-		if strings.ToLower(v) == domain {
+		g := glob.MustCompile(strings.ToLower(v))
```

Comment:
This could be out of loop or a global variable.

Reply:
We can change the first parameter of this function from `[]string` to `[]glob.Glob`.